### PR TITLE
Updates the U.S.S.P with new looks and loadouts for admins v2

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Partially reopens https://github.com/ParadiseSS13/Paradise/pull/16250, made by Stokes52 and being done with his permission.
Adds equipment and uniforms so that the U.S.S.P. can finally be used as a proper minor faction.

* New uniforms, jackets and hats, for Soviets in various ranks (see images)
* New and updated outfit lists for admins to easily spawn the new Soviets
* Soviet jobs will now show up on SecHUDs
* Mobs now receive Neo-Russkiya language when soviet outfit is equipped.
## Why It's Good For The Game

In theory, the soviets are a perfect candidate for fun and interesting events. They can be used in friendly, neutral or hostile roles depending on the context, but unfortunately their existing implementation left much to be desired in both form and function.

This PR was inspired in part by the recent SolGov improvements. Now, the Soviets have enough interesting equipment of their own to act as a balance or counter to SolGov. Variety is the spice of life, and If these changes are approved by maintainers and enjoyed by the players, it can lay a foundation for future expansions to the U.S.S.P.
## Images of changes

**Soviet Box:**
image

**Soviet Headset**
image

### **Outfit Lists:**
**Updated admin quick outfit selection for Soviets.**

Soviet Tourist
image
Mostly harmless. Probably visiting from a nearby station on holiday.

Soviet Conscript.
image
The last line of defense. Poorly armed with a bolt action rifle, a bayonet, some ammo, and a nagant revolver when all else fails. Would probably be pretty entertaining for the ghosts watching these guys try to fight anything.

Soviet Soldier
image
Generic soldier. Carries an APS automatic pistol and some ammo. Perfect for guard duty or more peaceful events.

Soviet Officer
image
For when you need someone to be in charge. Carries a classic russian .357 as his sidearm.

Soviet Admiral
image
No longer wears that awful pirate suit, he's now got his own fancy uniform!

## Testing

- Spawned multiple humans and gave them different soviet loadouts.
- Verified their hud icons.
- Verified loadout contents
## Changelog
:cl: Henri215, Stokes52 
add: New uniforms, coats and hats for Soviets
add: Soviet jobs now appear on the SecHUD
tweak: New admin outfit lists for easily spawning the updated Soviets
tweak: Mobs now receive Neo-Russkiya language when soviet outfit is equipped
/:cl:

### CODE REVIEW

**Comment by S34NW** [comment]
I like this, but your soviet box is not in the correct perspective. We updated all our boxes after the initial attempt at this PR, please update your box.

**Comment by Henri215** [comment]
> I like this, but your soviet box is not in the correct style (as all the other boxes are). We updated all our boxes after the initial attempt at this PR, please update your box.

I am aware but Stokes52 wont be able to update it, if i dont find someone to update it soon then it will be removed.

**Comment by Henri215** [comment]
Soviet box with better perspective has been added.

**Comment by Henri215** [comment]
Added soviet headset version, utilizes spec ops channel.

File: code/modules/clothing/suits/misc_suits.dm [hunk 1]
```diff
@@ -958,6 +958,23 @@
 		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
 		)
 
+/obj/item/clothing/suit/sovietcoat
+	name = "\improper Soviet greatcoat"
+	desc = "A military overcoat made of rough wool that is thick enough to provide excellent protection against the elements."
+	icon_state = "sovietcoat"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	armor = list(melee = 15, bullet = 15, laser = 15, energy = 5, bomb = 15, bio = 0, rad = 0, fire = 30, acid = 30)
```
Comment (Sirryan2002) [inline code/modules/clothing/suits/misc_suits.dm:966]: These need to use the armor defines (please see above usages)
such as:
```dm
list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 10, FIRE = 20, ACID = 40)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 2]
```diff
@@ -958,6 +958,23 @@
...
+	cold_protection = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	heat_protection = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	allowed = list(/obj/item/flashlight,/obj/item/gun/energy,/obj/item/gun/projectile,/obj/item/ammo_box)
+
+/obj/item/clothing/suit/sovietcoat/officer
+	name = "\improper Soviet officer's greatcoat"
+	desc = "A military overcoat made with expensive wool. The U.S.S.P armband means it must belong to someone important."
+	icon_state = "sovietofficercoat"
+	armor = list(melee = 25, bullet = 25, laser = 25, energy = 10, bomb = 20, bio = 0, rad = 0, fire = 30, acid = 30)
```
Comment (Sirryan2002) [inline code/modules/clothing/suits/misc_suits.dm:976]: same here

File: code/modules/clothing/suits/misc_suits.dm (hunk 1 above)
Comment (Henri215) [reply code/modules/clothing/suits/misc_suits.dm:966]: Done

**Comment by Henri215** [comment]
image

How the loadouts look with the new headset.

File: code/modules/clothing/suits/misc_suits.dm [hunk 3]
```diff
@@ -958,6 +958,23 @@
...
+/obj/item/clothing/suit/sovietcoat
+	name = "\improper Soviet greatcoat"
+	desc = "A military overcoat made of rough wool that is thick enough to provide excellent protection against the elements."
+	icon_state = "sovietcoat"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	armor = list(MELEE = 15, BULLET = 15, LASER = 15, ENERGY = 5, BOMB = 15, BIO = 0, RAD = 0, FIRE = 30, ACID = 30)
+	cold_protection = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	heat_protection = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	allowed = list(/obj/item/flashlight,/obj/item/gun/energy,/obj/item/gun/projectile,/obj/item/ammo_box)
```
Comment (Sirryan2002) [inline code/modules/clothing/suits/misc_suits.dm:970]:
```suggestion
	allowed = list(/obj/item/flashlight, /obj/item/gun/energy, /obj/item/gun/projectile, /obj/item/ammo_box)
```

File: code/game/jobs/access.dm [hunk 4]
```diff
@@ -400,6 +400,9 @@
 /proc/get_all_solgov_jobs()
 	return list("Solar Federation Lieutenant","Solar Federation Specops Lieutenant","Solar Federation Marine","Solar Federation Specops Marine","Solar Federation Representative","Sol Trader","Solar Federation General")
 
+/proc/get_all_soviet_jobs()
+	return list("Soviet Tourist","Soviet Conscript","Soviet Soldier","Soviet Officer","Soviet Admiral")
```
Comment (Sirryan2002) [inline code/game/jobs/access.dm:404]:
```suggestion
	return list("Soviet Tourist", "Soviet Conscript", "Soviet Soldier", "Soviet Officer", "Soviet Admiral")
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Pull request merged**